File: Practica-1-INF-121/INTRODUCCION POO/EJERCICIO-1/Python/Persona.py
#Crea una clase Persona con nombre, edad y ciudad
class Persona:
    def __init__ (self , nombre,edad,ciudad):
        self.nombre=nombre
        self.edad = edad
        self.ciudad = ciudad
    #a) Agrega un método para mostrar el saludo: “Hola, soy {nombre} de {ciudad}”
    def mostrarSaludo (self):
        print(f"Hola , soy {self.nombre} de {self.ciudad}.")
    #c) Agrega un método para verificar si es mayor de edad
    def es_mayor (self):
        return self.edad >= 18
#b) Crea tres personas y muestra su saludo
persona1 = Persona("Deysly", 18, "La Paz")
persona2 = Persona("Angel", 23, "Cochabamba")
persona3 = Persona("Ledys", 30, "Riberalta")

# Mostrar saludos
persona1.mostrarSaludo()
persona2.mostrarSaludo()
persona3.mostrarSaludo()

# Verificar si son mayores de edad
print(f"{persona1.nombre} es mayor de edad: {persona1.es_mayor()}")
print(f"{persona2.nombre} es mayor de edad: {persona2.es_mayor()}")
print(f"{persona3.nombre} es mayor de edad: {persona3.es_mayor()}") 
    

File: Practica-1-INF-121/INTRODUCCION POO/EJERCICIO-1/Java/Persona.java
public class Persona {

    public String nombre;
    public int edad ;
    public String ciudad;
    
    public Persona (String nombre, int edad, String ciudad){
        this.nombre=nombre;
        this.edad=edad;
        this.ciudad=ciudad;   
    }
    
    public void mostrarSaludo(){
        System.out.println("Hola, soy "+ nombre+ "de " + ciudad+".");
    }
    public boolean mayorDeEdad(){
        return edad >= 18;
    }
    public String getNombre(){
        return nombre;
    }
    public int getEdad(){
        return edad;
 
    }
    public String getCiudad(){
        return ciudad;
    }   
    
}
